docs/source/release_notes.rst

```diff
@@ -16,8 +16,9 @@ Release Notes
         * Added more information to users about ensembling behavior in ``AutoMLSearch`` :pr:`1527`
         * Add woodwork support for more utility and graph methods :pr:`1544`
         * Changed ``DateTimeFeaturizer`` to encode features as int :pr:`1479`
-        * Added `Linear Discriminant Analysis Transformer` component for dimensionality reduction :pr:`1331`
+        * Added ``LDA`` component for dimensionality reduction :pr:`1331`
```

Comment:
@bchen1116 that's a good point. I changed it for formatting purposes, because it used single-ticks instead of double-ticks. But instead I'll change the text back and remove the ticks :)